vendor/assets/javascripts/util.js

```diff
@@ -0,0 +1,7 @@
+Cmsify.util = {
+  filterAuthenticityToken: function(arr) {
+    return arr.filter(function(element) {
+      return element.name !== "authenticity_token";
+    });
+  }
+}
```

Comment:
Any reason why this shouldn't just be part of the default functionality of `serializeFields`?